lld/wasm/Config.h

```diff
@@ -173,6 +173,11 @@ struct Ctx {
     // Symbol whose value is the alignment of the TLS block.
     GlobalSymbol *tlsAlign;
 
+    // __rodata_start/__rodata_end
+    // Symbols marking the start/end of readonly data
+    DefinedData *rodataStart;
```

Comment:
Kind of unfortunate that we have `__stack_low`, `__heap_base`, and `__rodata_start`. But I guess it's too late to fix the inconsistency.